cmd/flux/cmd/test.go

```diff
@@ -207,10 +207,8 @@ func (t *Test) consume(ctx context.Context, results flux.ResultIterator) error {
 			err := result.Tables().Do(func(tbl flux.Table) error {
 				// The data returned here is the result of `testing.diff`, so any result means that
 				// a comparison of two tables showed inequality. Capture that inequality as part of the error.
-				// XXX: rockstar (08 Dec 2020) - This could use some ergonomic work, as the diff output
-				// is not exactly "human readable."
-				_, _ = fmt.Fprint(&output, table.Stringify(tbl))
```

Comment:
Should we remove the `Stringify` implementation if its now unused?

Reply:
Sadly it is not unused, it's still used in a bunch of tests and test utilities.